File: test_tts.py
import dashscope
from dashscope.audio.tts_v2 import SpeechSynthesizer

dashscope.api_key = 'sk-8c86fe89446848429434910a2fdb9d1a'
synthesizer = SpeechSynthesizer(model='cosyvoice-v1', voice='longxiaochun')
try:
    audio = synthesizer.call("你好，很高兴认识你")
    print("Result size:", len(audio))
except Exception as e:
    print("Error:", e)

File: TestTTS.java
import com.alibaba.dashscope.audio.ttsv2.SpeechSynthesisParam;
import com.alibaba.dashscope.audio.ttsv2.SpeechSynthesizer;
import java.nio.ByteBuffer;
public class TestTTS {
    public static void main(String[] args) throws Exception {
        SpeechSynthesisParam param = SpeechSynthesisParam.builder()
            .apiKey("sk-8c86fe89446848429434910a2fdb9d1a")
            .model("cosyvoice-v1")
            .voice("longxiaochun")
            .format("mp3")
            .build();
        SpeechSynthesizer synthesizer = new SpeechSynthesizer(param, null);
        ByteBuffer buffer = synthesizer.call("你好，这是一次测试。");
        System.out.println("Buffer null? " + (buffer == null));
    }
}
